docs/pywbemclicmdshelp.rst

```diff
@@ -882,27 +884,23 @@ The following defines the help output for the `pywbemcli connection save --help`
 
 ::
 
-    Usage: pywbemcli connection save [COMMAND-OPTIONS]
+    Usage: pywbemcli connection save [COMMAND-OPTIONS] NAME
 
       Save the current connection as a new WBEM connection definition.
 
       Create a new WBEM connection definition in the connections file from the
```

Comment:
Add "named NAME" to this sentence, i.e. "Create a new WBEM connection definition named NAME in the".